### ISSUE TITLE
is it still undetected ? 

### ISSUE THREAD

**Issue by eni9mv** [danielkrupinski/OneByteRadar#15]
11 oct 2022 ...

**Comment by PREEF1RE** [comment]
Yes, its not touching Cvar, its safe

**Comment by master-of-darkness** [comment]
are you sure?

**Comment by PREEF1RE** [comment]
Yes, its touching one byte in csgo. Vac Can always see the radar hack are turnt off , so its safe

**Comment by eni9mv** [comment]
> Yes, its touching one byte in csgo. Vac Can always see the radar hack are turnt off , so its safe

can u send me ur discord plz :) ?

**Comment by PREEF1RE** [comment]
PREEF1RE#3285

**Comment by JAREEEEK** [comment]
Is it possible to play FACEIT ? Will faceit anticheat detect it ?
